src/main/scala/net/cakesolutions/CakeBuildInfoPlugin.scala

```diff
@@ -75,7 +75,15 @@ object CakeBuildInfoPlugin extends AutoPlugin {
       }
     },
     packageOptions in (Compile, packageBin) +=
-      Package.ManifestAttributes(generalInfo.value.toSeq: _*)
+      Package.ManifestAttributes(generalInfo.value.toSeq: _*),
+    projectRoot := {
```

Comment:
Added so that we always can tell what our project root is - no more relative paths!

Reply:
Since it's based on Git, should we have logging in case a repo is not found? It'd make it easier to debug why `projectRoot` points to something you wouldn't expect it to.

Reply:
Logging is not possible here. Problem is that `streams.value` is a task and we are in a settings context here.